e2e/e2e.go

```diff
@@ -132,6 +132,8 @@ func run(ctx context.Context) error {
 		return fmt.Errorf("assert logs: %w", err)
 	}
 
+  // TODO(patrick.pichler): add assert for process tree events
```

Comment:
Are you going to do this as part of this PR or some other one?

Reply:
Added tests